test/v1alpha1/service.go

```diff
@@ -346,8 +347,16 @@ func IsServiceReady(s *v1alpha1.Service) (bool, error) {
 	return s.Generation == s.Status.ObservedGeneration && s.Status.IsReady(), nil
 }
 
-// IsServiceNotReady will check the status conditions of the service and return true if the service is
-// not ready.
+// IsServiceNotReady will check the status conditions of the service and return false if the service is
```

Comment:
Don't take take the suggestion straight from github as the line bleeds over, but use consistent style with below.

```suggestion
// IsServiceNotReady checks the Ready status condition of the service and returns true only if Ready is set to False.
```